rxconf/config_types.py

```diff
@@ -63,6 +67,10 @@ async def load_from_path_async(
     ) -> "FileConfigType":
         pass
 
+    @abstractmethod
+    def get_hash(self) -> int:
```

Comment:
Let's make this method private?

`__eq__` is enough to compare configs, `get_hash` is redundant.